dbt/include/athena/macros/utils/datediff.sql

```diff
@@ -0,0 +1,28 @@
+{% macro athena__datediff(first_date, second_date, datepart) -%}
```

Comment:
nit,  we need to find a way to test those macros

Reply:
Yes, that is my next hot topic to add some tests and these are great first candidates 👍🏻